saleor/plugins/openid_connect/plugin.py

```diff
@@ -246,12 +251,16 @@ def external_logout(self, data: dict, request: WSGIRequest, previous_value):
     def authenticate_user(self, request: WSGIRequest, previous_value) -> Optional[User]:
         if not self.active:
             return previous_value
-        # TODO this will be covered by tests and modified after we add Auth backend for
-        #  plugins
         token = get_token_from_request(request)
         if not token:
             return None
-        return get_user_from_access_token(token)
+        user = previous_value
```

Comment:
Let's remove this assignment to make code more readable in the if condition below.